3-12-2018/grzesiek/weather/Gemfile

```diff
@@ -0,0 +1,7 @@
+source 'https://rubygems.org'
+
+gem 'sinatra'
+gem 'rspec'
```

Comment:
Gems could be grouped for test and development

Reply:
Thanks for advice, I'll keep it in my mind for the future projects.